patches/chrome-browser-resources-settings-appearance_page-appearance_page.js.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/chrome/browser/resources/settings/appearance_page/appearance_page.js b/chrome/browser/resources/settings/appearance_page/appearance_page.js
+index 1c7e3474a595250da00bc9249a047e4e1eae1786..e585821e63703e66c9f06f928ec35713d15f570f 100644
+--- a/chrome/browser/resources/settings/appearance_page/appearance_page.js
++++ b/chrome/browser/resources/settings/appearance_page/appearance_page.js
+@@ -87,6 +87,20 @@ Polymer({
+       ],
+     },
+ 
++    // <if expr="not _google_chrome">
```

Comment:
I think we can include another js file here to do this. Talk to @yrliou because she did something similar

Reply:
example here https://github.com/brave/brave-core/commit/45c8a93e444a9668214c552a6b3ff86524deba7b

Reply:
brave_appearance_page.js is introduced.